src/fides/api/models/pre_approval_webhook.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey, String, Boolean
+from sqlalchemy.orm import relationship  # type: ignore
+from fides.api.db.base_class import Base
+from fides.api.models.connectionconfig import ConnectionConfig
+from fides.api.models.privacy_request import (
+    PrivacyRequest,
+)
+
+
+class PreApprovalWebhook(Base):
+    name = Column(String, unique=True, nullable=False)
+    key = Column(String, index=True, unique=True, nullable=False)
+    connection_config_id = Column(
+        String, ForeignKey(ConnectionConfig.id_field_path), nullable=False
+    )
+    connection_config = relationship(
+        ConnectionConfig, back_populates="pre_approval_webhook", uselist=False
+    )
```

Comment:
Getting this error, you likely need to set up the corresponding relationship on the ConnectionConfig since you're using `back_populates` - 
```
Error occurred writing config settings to database: One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class PreApprovalWebhook->preapprovalwebhook'. Original exception was: Mapper 'mapped class ConnectionConfig->connectionconfig' has no property 'pre_approval_webhook'
```

Reply:
yes good catch! I've added this now!

Reply:
there's often some tricky import issues to figure out around these!

Reply:
Maybe experiment with putting ConnectionConfig in quotes where the relationship is defined on the PreApprovalWebhook side -

Reply:
hm, that doesn't work either (locally when I run `nox -s collect_tests`). Very weird as the pattern is similar to how we've previously defined the `access_manual_webhook` relationship on `ConnectionConfig`

Reply:
this can be tricky! There's not a one-size-fits-all answer, sometimes I end up having to collocate models in the same file, sometimes I drop back into using "backref" (the old way) instead of "back_populates" which defines the relationship on one size instead of both.